proto/osmosis/cron/v1beta1/tx.proto

```
@@ -0,0 +1,69 @@
+syntax = "proto3";
+package osmosis.cron.v1beta1;
```

Buf (1.47.2)

Files with package "osmosis.cron.v1beta1" must be within a directory "osmosis/cron/v1beta1" relative to root but were in directory "proto/osmosis/cron/v1beta1".

(PACKAGE_DIRECTORY_MATCH)